### ISSUE TITLE
frequency calculation code update

### ISSUE THREAD

**Issue by s4il3sh** [ubsuny/23-Homework5G3#55]
I have sent a merge request for the updated code. Please accept them all @uarif. @LinxuanHu you are good to go now. Good luck!

**Comment by uarif** [comment]
Please confirm that I have merged your code. @s4il3sh  as you had multiple requests and I don't want to re merge your code, please check and let me know if I still need to merge your request or not? Because it has no description, Thank you

**Comment by s4il3sh** [comment]
> @s4il3sh Why did you choose the units per decades?

because I got less frequency so I had to increase time. But it was before fft now the code is updated with different frequency unit.

**Comment by s4il3sh** [comment]
> Please confirm that I have merged your code. @s4il3sh as you had multiple requests and I don't want to re merge your code, please check and let me know if I still need to merge your request or not? Because it has no description, Thank you

@uarif yes. thank you. One more merge request left. Yes, i sent different merge request for different file.
